src/status_im/contexts/profile/settings/list_items.cljs

```diff
@@ -35,7 +35,7 @@
      :blur?       true
      :action      :arrow}
     {:title       (i18n/label :t/wallet)
-     :on-press    #(rf/dispatch [:navigate-to-within-stack [:screen/settings.wallet :screen/settings]])
+     :on-press    #(rf/dispatch [:navigate-to-within-stack [:screen/settings.wallet :settings]])
```

Comment:
The settings screen wasn't using the `:screen/` qualifier 🤦. Fixed in this PR. https://github.com/status-im/status-mobile/pull/20364/commits/6a2bb51ee101d3affa193f669d240f0f90df0694